crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -4054,3 +4058,22 @@ fn get_cybersource_card_type(card_network: common_enums::CardNetwork) -> Option<
         common_enums::CardNetwork::Interac | common_enums::CardNetwork::RuPay => None,
     }
 }
+
+pub trait RemoveNewLine {
+    fn remove_new_line(&self) -> Self;
+}
+
+impl RemoveNewLine for Option<Secret<String>> {
+    fn remove_new_line(&self) -> Self {
+        self.clone().map(|masked_value| {
+            let new_string = masked_value.expose().replace("\n", " ");
+            Secret::new(new_string)
+        })
+    }
+}
+
+impl RemoveNewLine for Option<String> {
+    fn remove_new_line(&self) -> Self {
+        self.clone().map(|value| value.replace("\n", " "))
+    }
+}
```

Comment:
LGTM
Nit: This can be moved to connector utils

Reply:
Because of priority of fix, we can move ahead with this. Will be refactored in the next change